docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -270,6 +270,29 @@ realm password policy.
 * `Hashing Algorithm: pbkdf2-sha256`
 * `Hashing Iterations: 27500`
 
+= Keycloak CR cache-config-file option
+
+The Keycloak CR now allows for specifying the `cache-config-file` option via the `cache` spec `configMapFile` field, for example:
+
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  ...
+  cache:
+    configMapFile:
+      name: my-configmap
+      key: config.xml
+----
```

Comment:
I'd suggest moving this to the release notes as it's a new feature and not a breaking change (as polling vs. watching is).